lib/sql-string.js

```diff
@@ -44,7 +61,7 @@ function escape(val, timeZone, dialect, format) {
     if (dialect === 'postgres' && !format) {
       return dataTypes.ARRAY.prototype.stringify(val, {escape: partialEscape});
     }
-    return val.map(partialEscape);
```

Comment:
I believe this will break other dialects, MSSQL i think. Will probably need more tests

Reply:
I added integration test, looks like it's okay with MSSQL.

Reply:
I will learn. didn't sleep 3 days. thank you